rslib/src/storage/card/mod.rs

```diff
@@ -747,7 +747,7 @@ impl super::SqliteStorage {
             .get(0)?)
     }
 
-    pub(crate) fn get_costs_for_retention(&self) -> Result<(f32, f32, f32)> {
+    pub(crate) fn get_costs_for_retention(&self) -> Result<(f32, f32, f32, f32)> {
```

Comment:
For future maintenance, the return value would be better broken into a separate struct.